src/vstest.console/vstest.console.csproj

```diff
@@ -10,6 +10,7 @@
     <WarningsAsErrors>true</WarningsAsErrors>
     <OutputType>Exe</OutputType>
     <PlatformTarget Condition="'$(TargetFramework)' == 'net451'">AnyCPU</PlatformTarget>
+    <Prefer32Bit Condition="'$(TargetFramework)' == 'net451'">true</Prefer32Bit>
```

Comment:
Irrespective of this UWP support won't work in dotnet test, but with this we can atleast bring it up in vstest.console
I agree that we don't want to be specific to vstest.console, but right now I see no option
I'll update UWP team, to take consider that scenario as well, & try to port themselves to 64bit as well